test/index.js

```diff
@@ -43,17 +87,28 @@ describe('cron-allowed-range', function() {
       );
     });
 
+    ['0-59 0-23 1-31 1-11,12 0-6'].forEach(expression => {
+      it(`should not throw any error when the values are exactly on the range'${expression}'`, function() {
+        const cr = new CronAllowedRange(expression);
+        assert.sameDeepMembers(cr.minute, [{ start: 0, end: 59 }]);
+        assert.sameDeepMembers(cr.hour, [{ start: 0, end: 23 }]);
+        assert.sameDeepMembers(cr.dayOfMonth, [{ start: 1, end: 31 }]);
+        assert.sameDeepMembers(cr.month, [{ start: 1, end: 11 },{ start: 12, end: 12 }]);
+        assert.sameDeepMembers(cr.dayOfWeek, [{ start: 0, end: 6 }]);
+      });
+    });
```

Comment:
I think it would be valuable to test values right outside the range. E.g. `minute` with a value of `60`. What do you think?

Reply:
Yes will add them 👍